controllers/components/kserve/kserve_support.go

```diff
@@ -266,3 +267,14 @@ func ownedViaFT(cli client.Client) handler.MapFunc {
 		return []reconcile.Request{}
 	}
 }
+
+func isLegacyOwnerRef(or metav1.OwnerReference) bool {
+	switch {
+	case or.APIVersion == gvk.DataScienceCluster.GroupVersion().String() && or.Kind == gvk.DataScienceCluster.Kind:
+		return true
+	case or.APIVersion == gvk.DSCInitialization.GroupVersion().String() && or.Kind == gvk.DSCInitialization.Kind:
```

Comment:
should we handle the ones from DSCI? which is not serverless related

Reply:
this is copy an paste, I'll remove it

Reply:
done